winui/Kanban/events.md

```diff
@@ -0,0 +1,762 @@
+            taskDetail.Tags = new List<string>() { "New control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle3.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "New Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Dragging events support for Kanban";
+            taskDetail.Category = "Closed";
+            taskDetail.IndicatorColorKey = "Normal";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle4.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "WF Issue";
+            taskDetail.Id = "1254";
+            taskDetail.Description = "HorizontalAlignment for tooltip is not working";
+            taskDetail.Category = "Review";
+            taskDetail.IndicatorColorKey = "High";
+            taskDetail.Tags = new List<string>() { "Bug fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle5.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+            return taskDetails;
+        }
+
+        /// <summary>
+        /// Handles the event when a Kanban card is tapped.
+        /// </summary>
+        /// <param name="parameter"></param>
+        private void OnCardTapped(object parameter)
+        {
+            if (parameter is KanbanCardTappedEventArgs args)
+            {
+                var selectedCoumn = args.SelectedColumn;
+            }
+        }
+
+        /// <summary>
+        /// Determines whether card tapped can be executed.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <returns></returns>
+        private bool CanExecuteCardTapped(object sender)
+        {
+            return true;
+        }
+
+        #endregion
+    }
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="RelayCommand" %}
+
+public class RelayCommand : ICommand
+{
+    public event EventHandler CanExecuteChanged;
+
+    private readonly Action<object> execute;
+    
+    private readonly Predicate<object> canExecute;
+
+    public RelayCommand(Action<object> execute, Predicate<object> canExecute = null)
+    {
+        this.execute = execute;
+        this.canExecute = canExecute;
+    }
+
+    public RelayCommand(Action<object> executeAction)
+    {
+        this.execute = executeAction;
+    }
+
+    public bool CanExecute(object parameter) => canExecute == null || canExecute(parameter);
+
+    public void Execute(object parameter) => execute(parameter);
+
+
+    public void RaiseCanExecuteChanged()
```

Comment:
Have you tested this code? is it working properly?

Reply:
yes